src/components/_list-view.scss

```
@@ -700,3 +765,33 @@ $list-view-gaps: $gaps;
   box-sizing: border-box;
   padding-left: 1rem;
 }
+
+.ncgr-list-tile__checkbox {
+  display: none;
+}
+
+.ncgr-list-tile__checkbox ~ .ncgr-list-tile__hidden {
```

これが `expansion-body` かなぁ

```suggestion
.ncgr-list-tile__checkbox ~ .ncgr-list-tile__expansion-body {
```